src/main/java/com/bank/account/service/TransactionService.java

```diff
@@ -0,0 +1,17 @@
+package com.bank.account.service;
+
+import com.bank.account.mapper.domen.dto.CustomerRespDto;
+import com.bank.account.mapper.domen.dto.TransactionReqDto;
+import com.bank.account.mapper.domen.dto.TransactionRespDto;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
```

Comment:
Bad practice to have interfaces annotated with @Service. It will work, however, would be less understandable and readable.